builder-support/specs/pdns-recursor.spec

```diff
@@ -106,6 +106,9 @@ getent passwd pdns-recursor > /dev/null || \
 exit 0
 
 %post
+if [ ! -e %{_sysconfdir}/%{name}/recursor.conf -a ! -e %{_sysconfdir}/%{name}/recursor.yml ]; then
+    cp %{_sysconfdir}/%{name}/recursor.yml-dist %{_sysconfdir}/%{name}/recursor.yml
```

Comment:
The `cp` should probably be an install to make sure the perms and owner/group are right.